openstudiocore/src/gbxml/ReverseTranslator.cpp

```diff
@@ -623,6 +627,9 @@ namespace gbxml {
         surface.setSurfaceType("Wall");
       }else if (surfaceType.contains("UndergroundWall")){
         surface.setSurfaceType("Wall");
+        surface.setOutsideBoundaryCondition("Ground");
+        surface.setSunExposure("NoSun");
+        surface.setWindExposure("NoWind");
```

Comment:
@jasondegraw should we set boundary conditions for all cases?  how about for the other underground surfaces 'UndergroundCeiling', 'UndergroundSlab', 'SlabOnGrade', 'RaisedFloor', 'ExposedFloor'?  It looks like explicit boundary conditions from the xml would override these defaults on line 655 which is correct